controllers/usersignup/usersignup_controller.go

```diff
@@ -43,6 +50,12 @@ import (
 
 type StatusUpdaterFunc func(userAcc *toolchainv1alpha1.UserSignup, message string) error
 
+const (
+	migrationAnnotationName           = toolchainv1alpha1.LabelKeyPrefix + "migration-replaces"
+	migratedAnnotationName            = toolchainv1alpha1.LabelKeyPrefix + "migrated"
```

Comment:
what is the purpose of this annotation?

Reply:
This annotation is essentially informational, indicating to anyone looking at the UserSignup that it has been migrated.  After migration is complete for all users we can remove them.